File: composeApp/src/iosMain/kotlin/com/sample/project/utils/NativeConfig.swift
class NativeConfig {

    static func getBaseUrl() -> String {
        return ios_getBaseUrl()
    }

    static func getApiKey() -> String {
        return ios_getApiKey()
    }

    static func getClientId() -> String {
        return ios_getClientId()
    }

    static func getSecretKey() -> String {
        return ios_getSecretKey()
    }

    static func getAnotherKey() -> String {
        return ios_getAnotherKey()
    }
}

File: composeApp/src/androidMain/kotlin/com/sample/project/utils/NativeConfig.kt
package com.sample.project.utils

object NativeConfig {

    init {
        System.loadLibrary("native-lib")
    }

    external fun getBaseUrl(): String
    external fun getApiKey(): String
    external fun getClientId(): String
    external fun getSecretKey(): String
    external fun getAnotherKey(): String
}